### DISCUSSION TITLE
Cannot setup plugin

### DISCUSSION THREAD

**Discussion by aurkaxi** [nvim-tree/nvim-tree.lua#1691] [Q&A] [answered]
So i use vim-plug
my .config folder is like this

- autoload
- - plug.vim
- - plugins
- - - nvim-tree.lua
- - - vim-airline
- vim-plug
- - plugins.vim
- init.vim

Now plug.vim is the installation file for vim-plug. and plugins.vim goes like this:
```vim
call plug#begin('~/.config/nvim/autoload/plugged')

	Plug 'vim-airline/vim-airline'
	Plug 'nvim-tree/nvim-tree.lua'

call plug#end()
```

and my init.vim goes like this:
```vim
source $HOME/.config/nvim/vim-plug/plugins.vim
:set number
:set mouse=a
```

i do not understand where to put the [setup](https://github.com/nvim-tree/nvim-tree.lua#setup) section.

**Comment by gegoune** [comment, marked as answer]
Please have a read through https://github.com/nanotee/nvim-lua-guide on how to set up Lua in Neovim.

**Reply by aurkaxi** [reply]
On that repo i read "where to put lua", "Using Lua from Vimscript", "Existing tutorials for writing Lua in Neovim"
still couldn't make it.

**Reply by pleshevskiy** [reply]
```
...
call plug#end()

lua << EOF
-- PUT YOUR nvim-tree.lua config here
EOF
```

**Reply by aurkaxi** [reply]
@pleshevskiy life saver...
